controllers/user.js

```diff
@@ -337,6 +338,35 @@ async function getPreferences(req, res) {
 }
 
 
+async function getBreachStats(req, res) {
+  if (!req.token) {
+    return res.status(401).json({
+      errorMessage: "User breach stats requires an FXA OAuth token passed in the Authorization header.",
+    });
+  }
+  const fxaResponse = await FXA.verifyOAuthToken(req.token);
+  if (!fxaResponse) {
+    return res.status(404).json({
+      errorMessage: "Cannot find FXA for that OAuth token.",
+    });
+  }
+  const user = await DB.getSubscriberByFxaUid(fxaResponse.body.user);
+  if (!fxaResponse) {
```

Comment:
I think this is unreachable sine we check-n-bail above on L348.
Should this be `if (!user)`?

Reply:
Oh, yes. Duh! Good catch.